<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportSMIDialog</class>
 <widget class="QDialog" name="ImportSMIDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>484</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>656</width>
    <height>484</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import SMILES Molecules</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="openSmiButton">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Open Smiles list</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="5">
    <widget class="QLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Separator</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="separatorBox">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>32</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Blank</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TAB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>,</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="4" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>331</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Smiles at column</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="smileColumn">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Molecule name by</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="0">
       <widget class="QRadioButton" name="column_2">
        <property name="text">
         <string>By column</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="nameColumn">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="ordinalno">
        <property name="text">
         <string>Ordinal number</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="5" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Activity definition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>None</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Active</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Inactive/Decoys</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Set Label</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </item>
   <item row="4" column="4" colspan="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>279</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>261</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="cancelButton">
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QPushButton" name="importButton">
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>119</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Import</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
